src/rnp/rnpcfg.c

```diff
@@ -116,8 +118,12 @@ rnp_cfg_find(rnp_cfg_t *cfg, const char *key)
     return (i == cfg->count) ? -1 : (int) i;
 }
 
-/* resize keys/vals arrays to the new size. Only expanding is supported */
-static int
+/** @brief resize keys/vals arrays to the new size. Only expanding is supported now.
+ *  Pointers keys and vals will be freed in rnp_cfg_free
+ *
+ *  @return true on success, false if allocation fails.
```

Comment:
Actually rnp_cfg_free must be called on any cfg after it was created and initialized via rnp_cfg_init. Will update this function description.